R/codetrack.R

```diff
@@ -1,28 +1,32 @@
-.track_it_all <- function(rObjects, pObjects) {
-  
+.track_it_all <- function(rObjects, pObjects, se_name) {
+
   aobjs <- as.data.frame(rObjects$active_plots)
-  
+
   # cmds are kept only for the plots and not for the tables
   aobjs <- aobjs[aobjs$Type!="geneStat",]
 
   # storing to a text character vector
-  tracked_code <- c("## Here's the list of commands to generate the plots you created using iSEE",
-                    "## Just copy them in a live session of R where your SingleCellExperiment object is",
-                    "## ... and if you want to adjust something, just edit the corresponding lines!",
-                    "")
+  tracked_code <- c(
+    "## Here's the list of commands to generate the plots you created using iSEE",
+    "## Just copy them in a live session of R where your SingleCellExperiment object is",
+    "## ... and if you want to adjust something, just edit the corresponding lines!",
```

Comment:
Careful with the wording here; the edits will get overwritten upon closing the modal. We should be emphasizing read-only status where possible.

Reply:
I'll make it clearer then: the adjustments are to be thought once the user copies, executes, sees something needs updates, and edits the corresponding lines in his/her code *after* using `iSEE`

Reply:
Sorry for picking up on the comment later. I see now the line is gone anyway, and you updated that in a clean way